frontend/src/components/datasets/DatasetsTableColumns.tsx

```diff
@@ -84,15 +84,15 @@ export const DATASET_COLUMN_DEFINITIONS : {[key in DatasetColumnID] : DatasetCol
 		title: "Latest Release Status Update",
 		dataIndex: ['dataset', 'latest_release_update'],
 		render: (_, {dataset}) => {
-			return dataset.latest_release_update ? moment(dataset.latest_release_update).format("YYYY-MM-DD LT") : ""
+			return dataset.latest_release_update ? dayjs(dataset.latest_release_update).format("YYYY-MM-DD LT") : ""
```

Comment:
LT ? i think you can remove the format altogether with dayjs ... the output from that format contains the string LT which is not what is intended ... (LT for local time ? this is not something the user want to see)

Reply:
You sure users would prefer UTC time?

Reply:
It includes local time to the format yes. BUt yeah anyway, i'll make the time international then.